Modules/Core/Transform/include/itkResampleInPlaceImageFilter.h

```diff
@@ -0,0 +1,160 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef itkResampleInPlaceImageFilter_h
+#define itkResampleInPlaceImageFilter_h
+
+#include "itkImageToImageFilter.h"
+#include "itkVersorRigid3DTransform.h"
+
+namespace itk
+{
+/** \class ResampleInPlaceImageFilter
+ * \brief Resample an image in place.
+ *
+ * The current ITK resample image filter will generate a physical memory-
+ * modified version of the input image if the input transform is not identity. The
+ * abuse use of the filter can be cumbersome in a situation that the image is very
```

Comment:
I would maybe restate this last sentence: I find that "(...) can be cumbersome (...)" is not that informative about the problems that can arise from the abuse. Maybe saying that "(...) can be a source of problems. such as {mention a few of the issues that can arise}. For example, when the image is very large, and (...)".